vmsyscall/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "vmsyscall"
+version = "0.1.0"
+authors = ["Harald Hoyer <harald@redhat.com>"]
+license = "Apache-2.0"
+edition = "2018"
+
+categories = [
+    "no-std",
+]
+
+keywords = [
+    "errno",
+    "syscall",
+    "no_std",
+]
+
+[dependencies]
+serde = { version = "1.0", default-features = false, features = ["derive"] }
```

Comment:
I don't remember how cargo handles `no_std` in this case. But I think the consumer of this crate will also have to specify `no_std` as a feature for `serde`. This is just a note for my future recall. :)

Reply:
At least for serde_cbor it works this way. No_std only has to be defined by the implementing serialize/deserialize crate...

Reply:
Default is "std"
```
[features]
default = ["std"]

# Provide derive(Serialize, Deserialize) macros.
derive = ["serde_derive"]

# Provide impls for common standard library types like Vec<T> and HashMap<K, V>.
# Requires a dependency on the Rust standard library.
std = []
```